### PR TITLE
Added information about maximum payload size

### PR DESCRIPTION
**Pull request by ethouris**
People get often confused about the maximum payload size in SRT when trying to send RTP over SRT; this could be at least as a reference information what to be aware of in this case.

### CODE REVIEW

**Comment by jeandube** [comment]
Earlier versions of SRT had a default MSS of 1500 bytes and it has never been a subject of discussion.

**Comment by maxlovic** [comment]
`v.1.2.3` also has a payload size of 1316 bytes.
Please consider adding a reference to the constants `SRT_LIVE_DEF_PLSIZE` and `SRT_LIVE_MAX_PLSIZE` of `srt.h` to the doc file.

File: docs/live-streaming.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ transport unit size for SRT, and it is also often used when sending MPEG-TS over
 Note that SRT isn't limited to MPEG-TS -- it can be applied to any "live streaming" data 
 transmission (as long as you use Live mode, which is the SRT default mode). You can use 
 any other suitable data format, and any intermediate protocol on top of MPEG-TS with 
-an extra header (you still have an extra 140 bytes per unit for that purpose). 
+an extra header (if you want to do this - as people often try this with RTP - note
```
Comment (stevomatthews) [inline docs/live-streaming.md]:
```suggestion
an extra header (this is an option that people often try with RTP) - note
```

File: docs/live-streaming.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ transport unit size for SRT, and it is also often used when sending MPEG-TS over
...
+that 1316 is the default maximum payload size, which can be changed using
```
Comment (stevomatthews) [inline docs/live-streaming.md]:
```suggestion
that 1316 is the default maximum payload size, which can be changed using the
```

**Pull request merged**